backend/src/Edna/Dashboard/DB/Query.hs

```diff
@@ -135,19 +164,29 @@ getExperiments mProj mComp mTarget sorting pagination =
         )
       )
   where
-    sortingApp ((_experimentRec, _projectId, _methodologyId, uploadDate, _subExpId), _) =
+    eExperimentId' :: ExperimentRec -> SqlSerial Word32
+    eExperimentId' = eExperimentId
```

Comment:
Why do we need it?

Reply:
I was getting a weird compilation error involving complex Beam types without it :shrug: 

Let me check whether it's still necessary.

Reply:
No longer necessary, removed.